packages/astro/src/cli/index.ts

```diff
@@ -110,7 +112,10 @@ function resolveCommand(flags: Arguments): CLICommand {
 export async function cli(args: string[]) {
 	const flags = yargs(args);
 	const cmd = resolveCommand(flags);
-	const projectRoot = flags.projectRoot || flags._[3];
+	let projectRoot = flags.projectRoot;
+	if (!projectRoot && cmd !== 'add') {
```

Comment:
I don't think this project root behavior was ever documented, and I've never heard of anyone using it. Either add a note here to remove before v1.0, or go ahead and remove it now.

Reply:
I would put it under a `--project-root` flag

Reply:
Yep, removed.